### PR TITLE
Add FAQ to About page and make About page look much better

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

**Comment by hallaathrad** [comment]
About page is being worked on at #5667 and I'm involved.
Thanks for Font Awesome!!!

File: server/views/resources/about.jade [hunk 1]
```diff
@@ -1,171 +1,212 @@
...
+                    td.text-center
+                        i.fa.fa-comment
+                    td
+                        a(href="/stories") Stories from campers who've become professional developers
+
+            .spacer
+            h2.text-center Whom to contact for what:
+            table.table.table-stripe
+                tr
+                    td Support (team@freecodecamp.com)
```
Comment (SaintPeter) [inline server/views/resources/about.jade]: This should probably be a clickable `mailto:` link.

Comment (hallaathrad) [inline server/views/resources/about.jade]: do we want an exposed email? no spam deterrence this way

**Comment by SaintPeter** [comment]
I'm not sure about the "Our This Page", "Our That Page".  That line of "Ours" seems really repetitious and redundant.

Maybe a blanket "Find us in these media:" at the top and a list.

I worry that we're losing easy access to all the links.  If the page just looks like a FAQ, people might not scroll down.

I'm thinking that a table of contents at the top might make sense.  

I would also reorder the page as:
* TOC
* Stats
* Links
* FAQ
* Contacts

**Comment by hallaathrad** [comment]
Furthermore, marketing wise it could be beneficial to say it's theirs (the users).
And how about accordion-ing the faq?

**Comment by QuincyLarson** [comment]
@hallaathrad I considered an accordion style but it seemed like overkill. There aren't that many questions. And an accordion makes them unsearchable when they're collapsed.

A TOC could make sense.

Originally I didn't use "Our" but I decided to mix the social links in with the other links (our TOS, etc).

**Comment by QuincyLarson** [comment]
@hallaathrad @SaintPeter I went ahead and made the email address a mailto link. 
@BerkeleyTrue was also of the opinion that the page wasn't big enough to justify a table of contents.
If the FAQ gets bigger (hopefully not) we can consider creating an accordion or a TOC.

**Comment by SaintPeter** [comment]
I'm just concerned that important things that we want to share will be skipped because the page just looks like a FAQ.  It's not obvious that there is more below.  I think the links need to be above the FAQ.

**Pull request merged**